shipit.yml

```diff
@@ -15,3 +15,7 @@ ci:
 dependencies:
   post:
     - yarn run build: {timeout: 1800}
+
+deploy:
+  override:
+    - node_modules/.bin/lerna publish from-git --yes
```

Comment:
This is temporary and will need to be deployed via a throw-away release branch and shipit stack until we get shipit fixed.

Reply:
I'm pretty ok with this being merged to master, so quilt uses `from-git` for releases for a bit. That seems better than "deploys from master are blocked'